### PR TITLE
DIV-6516: Display general orders document

### PR DESCRIPTION
**Pull request by Dizrupto**
# Description

Make General orders display on the DN Frontend.

Fixes https://tools.hmcts.net/jira/browse/DIV-6516

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

**Test Configuration**:

* Hardware:
* O/S and version:
* JDK:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by madjava — APPROVED** [review]

File: test/unit/steps/petitionerProgressBar.test.js [hunk 1]
```diff
@@ -813,7 +859,9 @@ describe(modulePath, () => {
         'rejectionDnRefusalOrder',
         'deemedAsServedGranted',
         'dispenseWithServiceGranted',
-        'aosOverdueCoverLetter'
+        'aosOverdueCoverLetter',
+        'generalOrder',
+        'generalOrder'
```
Comment (yarpo) [inline test/unit/steps/petitionerProgressBar.test.js:864]: are you sure `generalOrder` should be twice?

**Review by yarpo — APPROVED** [review]

File: test/unit/steps/done.test.js [hunk 2]
```diff
@@ -158,13 +158,48 @@ describe(modulePath, () => {
...
+        'dnAnswers',
+        'rejectionDnRefusalOrder',
+        'clarificationDnRefusalOrder',
+        'coRespondentAnswers',
+        'respondentAnswers',
+        'deemedAsServedGranted',
+        'dispenseWithServiceGranted',
+        'aosOverdueCoverLetter',
+        'generalOrder',
+        'generalOrder'
```
Comment (jacmurphy) [inline test/unit/steps/done.test.js:199]: Is this an accidental duplication?

File: test/unit/steps/petitionerProgressBar.test.js (hunk 1 above)
Comment (jacmurphy) [reply test/unit/steps/petitionerProgressBar.test.js:864]: This is the second duplication. 
The other place is - test/unit/steps/done.test.js

File: test/unit/steps/done.test.js (hunk 2 above)
Comment (jacmurphy) [reply test/unit/steps/done.test.js:199]: No, its in regards to the test data above that puts in two files. This is needed.

File: test/unit/steps/petitionerProgressBar.test.js (hunk 1 above)
Comment (jacmurphy) [reply test/unit/steps/petitionerProgressBar.test.js:864]: No, its in regards to the test data above that puts in two files. This is needed.

**Pull request merged**